.github/workflows/continuous-integration.yaml

```diff
@@ -2,13 +2,19 @@
 name: Continuous Integration
 
 # runs on
-# * pushes and pull requests on the "main"
+# * pushes and pull requests on the "main" (pull request only for specific paths)
```

Comment:
why wouldn't we apply the path filters to the push trigger as well?

Reply:
We could, I just like that there is a clear check-mark on each commit to main.. Should I change?

Reply:
Ok so with this configuration, the workflow would run on each PR that has main as its target (for each commit on the branch of the PR, I guess), and again when the PR is merged to main? If so I think that is fine.

Reply:
* runs always we push something on main (basically merge a PR)
* runs in PRs for commits that have source code changes

OK?

Reply:
yes, I approved PR